tests/shared.py

```diff
@@ -186,10 +186,14 @@ def run(self, *command):
 
 
 class GitRepo(Repo):
-    def __init__(self, content_dir):
+    def __init__(self, content_dir, **init_default_branch):
         super().__init__(content_dir)
 
         self.run('git', 'init')
```

Comment:
I think the best approach will be to use the `--initial-branch` flag here as part of the `git init` command, in the case where `init_default_branch` is set. Python has some special syntax for "include all the elements of this list as positional arguments", so the call might end up looking like:

```python
branch_args = []
if init_default_branch is not None:
    branch_args = ["--branch", init_default_branch]
# Note the * character. This has no effect if the list is empty.
self.run('git', 'init', *branch_args)
```

Reply:
Looks nicer indeed. 
But just a fix, the option would be "--initial-branch" or "-b" for short.